app/views/providers/check_provider_answers/index.html.erb

```diff
@@ -0,0 +1,127 @@
+        </dd>
+        <dd class="app-check-your-answers__change">
+          <a href="#">
+            Change<span class="govuk-visually-hidden"><%= t '.section_2.first_name' %></span>
+          </a>
+        </dd>
+      </div>
+      <div class="app-check-your-answers__contents">
+        <dt class="app-check-your-answers__question">
+          <%= t '.section_2.last_name' %>
+        </dt>
+        <dd class="app-check-your-answers__answer">
+          <%= @applicant.last_name %>
+        </dd>
+        <dd class="app-check-your-answers__change">
+          <a href="#">
+            Change<span class="govuk-visually-hidden"><%= t '.section_2.last_name' %></span>
+          </a>
+        </dd>
+      </div>
+      <div class="app-check-your-answers__contents">
+        <dt class="app-check-your-answers__question">
+          <%= t '.section_2.dob' %>
+        </dt>
+        <dd class="app-check-your-answers__answer">
+          <%= @applicant.date_of_birth.strftime('%d/%m/%Y') %>
+        </dd>
+        <dd class="app-check-your-answers__change">
+          <a href="#">
+            Change<span class="govuk-visually-hidden"><%= t '.section_2.dob' %></span>
+          </a>
+        </dd>
+      </div>
+      <div class="app-check-your-answers__contents">
+        <dt class="app-check-your-answers__question">
+          <%= t '.section_2.nino' %>
+        </dt>
+        <dd class="app-check-your-answers__answer">
+          <%= @applicant.national_insurance_number %>
+        </dd>
+        <dd class="app-check-your-answers__change">
+          <a href="#">
+            Change<span class="govuk-visually-hidden"><%= t '.section_2.nino' %></span>
+          </a>
+        </dd>
+      </div>
+      <div class="app-check-your-answers__contents">
+        <dt class="app-check-your-answers__question">
+          <%= t '.section_2.address' %>
+        </dt>
+        <dd class="app-check-your-answers__answer">
+          <%= @address %>
+        </dd>
+        <dd class="app-check-your-answers__change">
+          <a href="#">
+            Change<span class="govuk-visually-hidden"><%= t '.section_2.address' %></span>
+          </a>
+        </dd>
+      </div>
+      <div class="app-check-your-answers__contents">
+        <dt class="app-check-your-answers__question">
+          <%= t '.section_2.email' %>
+        </dt>
+        <dd class="app-check-your-answers__answer">
+          <%= @applicant.email %>
+        </dd>
+        <dd class="app-check-your-answers__change">
+          <a href="#">
+            Change<span class="govuk-visually-hidden"><%= t '.section_2.email' %></span>
+          </a>
+        </dd>
+      </div>
+    </dl>
+
+    <h2 class="govuk-heading-m">
+      <%= t '.section_3.heading' %>
+    </h2>
+
+    <p>
+      <%= t '.section_3.text' %>
+    </p>
+
+  </div>
+</div>
+
+<%= button_to 'Submit', confirm_providers_legal_aid_application_check_provider_answers_path(@legal_aid_application), class: 'govuk-button' %>
```

Comment:
As you're using translations for everything else - why not here and for the back button too! That is, both the word "Submit" and "Back" should be defined via a translation. I'd suggest adding them to the `generic` section: `t 'generic.back'` for example.